server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -145,6 +145,20 @@ public CompletableFuture<Token> deleteToken(ServiceRequestContext ctx,
                             .thenApply(unused -> token.withoutSecret()));
     }
 
+    /**
+     * DELETE /tokens/{appId}/removed
+     *
+     * <p>Purges a token of the specified ID that was deleted before.
+     */
+    @Delete("/tokens/{appId}/deleted")
```

Comment:
This API should be allowed only for administrators.
```suggestion
    @Delete("/tokens/{appId}/deleted")
    @RequiresAdministrator
```